test/schema.sql

```diff
@@ -106,9 +106,12 @@ CREATE TABLE follows (
   ipfs VARCHAR(64) NOT NULL,
   follower VARCHAR(64) NOT NULL,
   space VARCHAR(64) NOT NULL,
+  network VARCHAR(24) NOT NULL DEFAULT 's',
```

Comment:
Not sure how we can handle it in schema 🤔 maybe we should not default it to `s` but update values later

Reply:
Leaving the field blank, then updating it later may result in duplicate field, as we need to update the primary index also, to include the new `network` field.

We could leave the schema as-is, and use the query in the PR description to set a different default value for different env.